Fix the CullPrimitiveEXT VUID's (#2475)
KhronosGroup/Vulkan-Docs: chapters/interfaces.adoc

    The variable decorated with code:CullPrimitiveEXT must: be declared
    using the code:Output {StorageClass}
  * [[VUID-{refpage}-CullPrimitiveEXT-07036]]
    code:CullPrimitiveEXT must: be declared as a boolean value member of a structure or a variable of type code:OpTypeArray of boolean values. 

>   * If code:CullPrimitiveEXT declared as an array of boolean values. The size of the array 

Reviewer: ```suggestion
  * If code:CullPrimitiveEXT declared as an array of boolean values, the size of the array 
```

>     code:CullPrimitiveEXT must: be declared as a boolean value member of a structure or a variable of type code:OpTypeArray of boolean values. 
>   * If code:CullPrimitiveEXT declared as an array of boolean values. The size of the array 
>     decorated with code:CullPrimitiveEXT must: match the value specified by code:OutputPrimitivesEXT
>   * If code:CullPrimitiveEXT is a member of a structure, there must: be a variable declared 
>     which is array of parent strucure that contains code:CullPrimitiveEXT. The size of the array must: match the value specified by code:OutputPrimitivesEXT.
>   * There must be only one declaration of the code:CullPrimitiveEXT associated with a entry
>     point's interface.

Reviewer: ```suggestion
    code:CullPrimitiveEXT must: decorate a scalar boolean member of a structure decorated as code:Block, or decorate a variable of type code:OpTypeArray of boolean values. 
  * If code:CullPrimitiveEXT is declared as an array of boolean values, the size of the array 
    must: match the value specified by code:OutputPrimitivesEXT
  * If code:CullPrimitiveEXT decorates a member of a structure, the variable declaration of the containing code:Block type must: have an array size that matches the value specified by code:OutputPrimitivesEXT
  * There must be only one declaration of the code:CullPrimitiveEXT associated with a entry
    point's interface.
```

Reviewer: I think this language reads closer to the rest of the spec, but otherwise, the gist of it is fine.

Author: Thanks for the suggestion, I am now adding same langugage for rest of the builtins.

    using the code:Output {StorageClass}
  * [[VUID-{refpage}-CullPrimitiveEXT-07036]]
    The variable decorated with code:CullPrimitiveEXT must: be declared as

>     a boolean value.

Reviewer 2: no period on the end of VUs

Author: updated the commit

>     a scalar of boolean value.

Reviewer 2: ```suggestion
    a boolean value
```

Adjust language to be similar to other builtins (eg HelperInvocation)

    {ExecutionModel} must: be declared using the code:Input {StorageClass}
  * [[VUID-{refpage}-Layer-04276]]
    The variable decorated with code:Layer must: be declared as a scalar
    32-bit integer value for all supported execution models except code:MeshEXT.
  * [[VUID-{refpage}-Layer-07039]]
    The variable decorated with code:Layer within the code:MeshEXT
    {ExecutionModel} must: also be decorated with the code:PerPrimitiveEXT
    decoration

>   * code:Layer within the MeshEXT Execution Model must: decorate a scalar 

Reviewer 2: ```suggestion
  * code:Layer within the code:MeshEXT {ExecutionModel} must: decorate a scalar 
```

    {ExecutionModel} must: be declared using the code:Input {StorageClass}
  * [[VUID-{refpage}-Layer-04276]]
    The variable decorated with code:Layer must: be declared as a scalar
    32-bit integer value for all supported execution models except code:MeshEXT.
  * [[VUID-{refpage}-Layer-07039]]
    The variable decorated with code:Layer within the code:MeshEXT
    {ExecutionModel} must: also be decorated with the code:PerPrimitiveEXT
    decoration

>   * code:Layer within the MeshEXT Execution Model  must: decorate a scalar 

Reviewer: ```suggestion
  * code:Layer within the MeshEXT Execution Model must: decorate a scalar 
```

    {StorageClass}
  * [[VUID-{refpage}-PrimitiveId-04337]]
    The variable decorated with code:PrimitiveId must: be declared as a
    scalar 32-bit integer value for all supported execution models except code:MeshEXT
  * [[VUID-{refpage}-PrimitiveId-07040]]
    The variable decorated with code:PrimitiveId within the code:MeshEXT
    {ExecutionModel} must: also be decorated with the code:PerPrimitiveEXT
    decoration

>   * code:PrimitiveId within the MeshEXT Execution Model must: decorate a 

Reviewer 2: ```suggestion
  * code:PrimitiveId within the code:MeshEXT {ExecutionModel} must: decorate a 
```

    {StorageClass}
  * [[VUID-{refpage}-PrimitiveId-04337]]
    The variable decorated with code:PrimitiveId must: be declared as a
    scalar 32-bit integer value for all supported execution models except code:MeshEXT
  * [[VUID-{refpage}-PrimitiveId-07040]]
    The variable decorated with code:PrimitiveId within the code:MeshEXT
    {ExecutionModel} must: also be decorated with the code:PerPrimitiveEXT
    decoration

>   * code:PrimitiveId within the MeshEXT Execution Model  must: decorate a 

Reviewer: ```suggestion
  * code:PrimitiveId within the MeshEXT Execution Model must: decorate a 
```

    enumerants in the SPIR-V specification
  * [[VUID-{refpage}-PrimitiveShadingRateKHR-07059]]
    The variable decorated with code:PrimitiveShadingRateKHR within the
    code:MeshEXT {ExecutionModel} must: also be decorated with the code:PerPrimitiveEXT 
    decoration

>   * code:PrimitiveShadingRateKHR within the MeshEXT Execution Model must: decorate 

Reviewer 2: ```suggestion
  * code:PrimitiveShadingRateKHR within the code:MeshEXT {ExecutionModel} must: decorate 
```

    enumerants in the SPIR-V specification
  * [[VUID-{refpage}-PrimitiveShadingRateKHR-07059]]
    The variable decorated with code:PrimitiveShadingRateKHR within the
    code:MeshEXT {ExecutionModel} must: also be decorated with the code:PerPrimitiveEXT 
    decoration

>   * code:PrimitiveShadingRateKHR within the MeshEXT Execution Model  must: decorate 

Reviewer: ```suggestion
  * code:PrimitiveShadingRateKHR within the MeshEXT Execution Model must: decorate 
```

    {ExecutionModel} must: be declared using the code:Input {StorageClass}
  * [[VUID-{refpage}-ViewportIndex-04408]]
    The variable decorated with code:ViewportIndex must: be declared as a
    scalar 32-bit integer value for all supported execution models except 
    code:MeshEXT
  * [[VUID-{refpage}-ViewportIndex-07060]]
    The variable decorated with code:ViewportIndex within the code:MeshEXT
    {ExecutionModel} must: also be decorated with the code:PerPrimitiveEXT
    decoration

>   * code:ViewportIndex within the MeshEXT Execution Model must: decorate a 

Reviewer 2: ```suggestion
  * code:ViewportIndex within the code:MeshEXT {ExecutionModel} must: decorate a 
```

    {ExecutionModel} must: be declared using the code:Input {StorageClass}
  * [[VUID-{refpage}-ViewportIndex-04408]]
    The variable decorated with code:ViewportIndex must: be declared as a
    scalar 32-bit integer value for all supported execution models except 
    code:MeshEXT
  * [[VUID-{refpage}-ViewportIndex-07060]]
    The variable decorated with code:ViewportIndex within the code:MeshEXT
    {ExecutionModel} must: also be decorated with the code:PerPrimitiveEXT
    decoration

>   * code:ViewportIndex within the MeshEXT Execution Model  must: decorate a 

Reviewer: ```suggestion
  * code:ViewportIndex within the MeshEXT Execution Model must: decorate a 
```
